tiledb/sm/query/readers/sparse_index_reader_base.h

```diff
@@ -330,14 +529,19 @@ class SparseIndexReaderBase : public ReaderBase {
    */
   ReadState* read_state();
 
+  /**
+   * Initializes the reader.
+   *
+   * @param skip_checks_serialization Skip checks during serialization.
+   */
+  void init(bool skip_checks_serialization);
```

Comment:
I'm not seeing an implementation of this method. Am I blind or was this meant to be removed.